boards/ti/sk_am62/board.cmake

```diff
@@ -0,0 +1,13 @@
+# Texas Instruments Sitara AM62x-SK-M4 EVM
+#
+# Copyright (c) 2024, PHYTEC Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_SOC_AM6234_M4)
+  board_set_debugger_ifnset(openocd)
+  board_set_flasher_ifnset(openocd)
+
+  board_runner_args(openocd "--no-init" "--no-halt" "--no-targets" "--gdb-port-offset=6")
```

Comment:
Good work!
We have done something similar on A53 cores, although not integrated into zephyr/west.
What do you think about specifying the port rather than an offset? e.g:
`--gdb-port=3333 --gdb-client-port=3339`

Reply:
yeah, that's actually not a bad idea. Thanks!